certs/postgres/certs.go

```diff
@@ -170,6 +195,77 @@ func (cr certsRepository) RetrieveBySerial(ctx context.Context, ownerID, serialI
 	return c, nil
 }
 
+func (cr certsRepository) ListExpiredCerts(ctx context.Context, timeBefore time.Duration, limit, offset uint64) (certs.Page, error) {
+	whereClause := fmt.Sprintf("WHERE expire <= now()  - interval '%s'", timeBefore)
```

Comment:
Are you sure using Database server time is better than using service time? It's possible that those two are not the same and this can be a source of the sneaky errors.

Reply:
Okay I agree with your point, I will change the approch

Reply:
This one doesn't seem addressed.

Reply:
Done, expire calculate from service time